src/components/IconSetPreviewFooter.tsx

```diff
@@ -72,7 +72,7 @@ const IconSetPreviewFooter = ({
     <>
       <div className="min-h-20 z-10 flex flex-col items-center justify-between gap-3 divide-neutral-300 bg-neutral-100 p-4 dark:bg-neutral-800 sm:flex-row">
         <div className="text-xs text-neutral-500">
-          {`${icons.length} icons`}
+          {`${icons.length} ${icons.length === 1 ? "icon" : "icons"}`}
```

Comment:
This condition could be better.

`icons.length > 1 ? 'icons' : 'icon'`

Reply:
Yes! I've just committed the fix